backend/functions/healthcheck/healthcheck.test.ts

```diff
@@ -33,3 +33,11 @@ test('Healthcheck endpoint should return an ALIVE status', async () => {
 
   expect(context.res.body).not.toBeNull(); // Check for any response.
 }, 10000);
+
+test('checkResults should return false when one result is false and true when all results are true', () => {
+  expect(checkResults(false, true, true, true)).toBe(false);
+  expect(checkResults(true, false, true, true)).toBe(false);
+  expect(checkResults(true, true, false, true)).toBe(false);
+  expect(checkResults(true, true, true, false)).toBe(false);
+  expect(checkResults(true, true, true, true)).toBeTruthy();
```

Comment:
I'm curious why we use `toBe(false)` and `toBeTruthy()` rather than both using a similar approach. Why not use `toBe(true)` or use `toBeFalsy()` to align all the expects?

Reply:
Yeah good point.